v63004/outref/gtm8900.txt

```diff
@@ -0,0 +1,29 @@
+# Create a single region DB with region DEFAULT
+# Check the DB
+
+# Testing MUPIP SET -[NO]ENCRYPT in a properly set up environment
+# --------------------------------------------------------------------------
+# \##SOURCE_PATH##/mupip set -ENCRYPTABLE -REGION 'DEFAULT'
+Database file ##TEST_PATH##/mumps.dat now has encryptable flag set to  TRUE
+
+# \##SOURCE_PATH##/mupip set -NOENCRYPTABLE -REGION 'DEFAULT'
+Database file ##TEST_PATH##/mumps.dat now has encryptable flag set to FALSE
+
+
+# Testing MUPIP SET -ENCRYPT  with no set gtm_passwd (expecting failure)
```

Comment:
ENCRYPT -> [NO]ENCRYPT
failure -> failure for ENCRYPT and success for NOENCRYPT